script/build.ts

```diff
@@ -144,6 +144,10 @@ function packageApp() {
       return targetArch
     }
 
+    if (targetArch === 'arm') {
+      return 'armv7l'
+    }
+
```

Comment:
can't comment high enough. look up a few lines, in the file I refererenced to you. you need to change
```
 if (
      targetArch === 'arm64' ||
      targetArch === 'x64' ||
      targetArch === 'armv7l'
    ) {
      return targetArch
    }
```
its input is `npm_config_arch` and armv7l isn't valid input. arm is but the returned value needs to be armv7l (for electron packager)

Reply:
actually nvm. the functionality as it is will work, it will just be impossible to hit `targetArch === 'armv7l'` since `npm_config_arch` should never contain that and `os.arch` will never return that